java/java-guestbook/skaffold.yaml

```diff
@@ -1,44 +1,14 @@
-# To learn more about the skaffold.yaml schema visit
-# https://skaffold.dev/docs/references/yaml/
-apiVersion: skaffold/v2beta4
+# The Guestbook app is configured to use skaffold modules.
+# The frontend and backend services can be activated by name,
+# allowing each service to be iterated on independently.
+# To learn more about skaffold modules and multi-config support,
+# visit https://skaffold.dev/docs/design/config/
+
+apiVersion: skaffold/v2beta18
 kind: Config
 build:
-  tagPolicy:
-    sha256: {}
-  # defines where to find the code at build time and where to push the resulting image
-  artifacts:
-    - image: java-guestbook-backend
-      # To learn more about how Jib builds Java containers visit
-      # https://github.com/GoogleContainerTools/jib
-      jib:
-        project: backend
-    - image: java-guestbook-frontend
-      jib:
-        project: frontend
-# defines the Kubernetes manifests to deploy on each run
-deploy:
-  kubectl:
-    manifests:
-    - ./kubernetes-manifests/*.service.yaml
-    - ./kubernetes-manifests/*.deployment.yaml
-portForward:
-  # Cloud Code automatically port forwards resources,
-  # or users can define explicit port forwards here.
-  # For more info, visit https://skaffold.dev/docs/pipeline-stages/port-forwarding/#UDPF
-  - resourceType: service
-    resourceName: java-guestbook-frontend
-    port: 80
-    localPort: 4503
-profiles:
-# use the cloudbuild profile to build images using Google Cloud Build
-- name: cloudbuild
-  build:
-    googleCloudBuild: {}
-# use the dockerfile profile to build images using Docker instead of Jib
-- name: dockerfile
-  build:
-    artifacts:
-      - image: java-guestbook-backend
-        context: backend
-      - image: java-guestbook-frontend
-        context: frontend
+  local:
+    concurrency: 0
+requires:
+  - path: ./frontend
+  - path: ./backend
```

Comment:
looks like there is a few missing newlines across all files, I think general standard is to have them at the end of tetx files

Reply:
Fixed across all files ✅